content/en/docs/tutorials/hello-minikube.md

```diff
@@ -55,10 +55,26 @@ If you installed minikube locally, run `minikube start`.
     minikube dashboard
     ```
 
+{{< note >}}
```

Comment:
Hi @CKchen0726 . Thanks for contributing!
You could indent the `note` shortcode under the second list item and rephrase these sentences:

The `dashboard` command enables the dashboard add-on and opens the proxy in the default web browser. You can create Kubernetes resources on the dashboard such as Deployment and Service.

If you are running in an environment as root, see [Open Dashboard with URL](/docs/tutorials/hello-minikube#open-dashboard-with-url).

To stop the proxy, run `Ctrl+C` to exit the process. The dashboard remains running.